index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+    <html lang="en">
```

Comment:
`Awesome`

Nice job using `lang="en"` (or variant) to the `html` start tag, this is good for internationalisation. You should check this [link for more info](https://www.w3.org/International/questions/qa-html-language-declarations#basics).  :+1: :star2: